loco-gen/src/template_engine.rs

```diff
@@ -0,0 +1,379 @@
+use std::collections::HashMap;
```

Comment:
if this file is just about helpers for views (scaffold etc) i would call it tera_scaffold_ext.rs
if its general extensions then:
tera_ext.rs

it's ok to name our engine explicitly (tera) just for the purpose of finding exactly what you're looking for "where is this tera builtin coming from? ah, tera_ext.rs"